portal/src/test/scala/cromwell/pipeline/utils/auth/TestUserUtils.scala

```diff
@@ -11,15 +11,54 @@ object TestUserUtils {
   def getDummyUser(password: String = userPassword, passwordSalt: String = "salt", active: Boolean = true): User = {
     val uuid = UUID.randomUUID().toString
     val passwordHash = StringUtils.calculatePasswordHash(password, passwordSalt)
+    createDummyUser(
+      UserId(uuid),
+      s"JohnDoe-$uuid@cromwell.com",
+      passwordHash,
+      passwordSalt,
+      "FirstName",
+      "LastName",
+      active
+    )
+  }
+
+  def getDummyUserWithCustomEmailDomain(
+    password: String = userPassword,
+    passwordSalt: String = "salt",
+    active: Boolean = true,
+    emailDomain: String
+  ): User = {
+    val uuid = UUID.randomUUID().toString
+    val passwordHash = StringUtils.calculatePasswordHash(password, passwordSalt)
+    createDummyUser(
+      UserId(uuid),
+      s"JohnDoe-$uuid@$emailDomain",
+      passwordHash,
+      passwordSalt,
+      "FirstName",
+      "LastName",
+      active
+    )
+  }
```

Comment:
This is basically a ctrl+c ctrl+v from `getDummyUser` method. Can you refactor it somehow so one of this method uses another?

Reply:
removed, done